xchange-gdax/src/main/java/info/bitrich/xchangestream/gdax/GDAXStreamingService.java

```diff
@@ -66,15 +71,15 @@ protected String getChannelNameFromMessage(JsonNode message) {
 
     @Override
     public String getSubscribeMessage(String channelName, Object... args) throws IOException {
-        GDAXWebSocketSubscriptionMessage subscribeMessage = new GDAXWebSocketSubscriptionMessage(SUBSCRIBE, product);
+        GDAXWebSocketSubscriptionMessage subscribeMessage = new GDAXWebSocketSubscriptionMessage(SUBSCRIBE, product, authData);
         ObjectMapper objectMapper = new ObjectMapper();
         return objectMapper.writeValueAsString(subscribeMessage);
     }
 
     @Override
     public String getUnsubscribeMessage(String channelName) throws IOException {
         GDAXWebSocketSubscriptionMessage subscribeMessage =
-                new GDAXWebSocketSubscriptionMessage(UNSUBSCRIBE, new String[]{"level2", "matches", "ticker"});
+                new GDAXWebSocketSubscriptionMessage(UNSUBSCRIBE, new String[]{"level2", "matches", "ticker"}, authData);
```

Comment:
Should this unsubscribe message also include "user" if `authData != null`?  Not seen any side-effects myself in use but it seems symmetrical.